packages/utils/src/lib/git.integration.test.ts

```diff
@@ -16,3 +24,66 @@ describe('getLatestCommit', () => {
     );
   });
 });
+
+describe('branchHasChanges', () => {
+  it('should return true if some changes are given', async () => {
+    await makeStatusDirty();
+    await expect(branchHasChanges()).resolves.toBe(true);
+    await makeStatusClean();
```

Comment:
Don't put test clean up after the assertion, it won't be run if the test fails, which would then affect other tests (which is bad, tests should be independent of each other). Use the `afterEach` hook.

Reply:
done